v2/specification.md

```diff
@@ -276,6 +279,19 @@ A POST request should be made to the payment protocol url.
 }
 ```
 
+* *weightedSize* is the length of the signed transaction in bytes, or the transaction "weight" in the Bitcoin blockchain.
```

Comment:
Thanks for adding these descriptions!

Reply:
yeah the "weightedSize" is pretty confusing for people who are not familiar with what segwit is doing